MapboxNavigation/NavigationMapView.swift

```diff
@@ -408,6 +408,36 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
     }
     
     // MARK: Feature Addition/Removal
+    /**
+     Showcases route array. Adds routes and waypoints to map, and sets camera to point encompassing the route.
+    */
+    public static let defaultPadding: UIEdgeInsets = UIEdgeInsets(top: 10, left: 20, bottom: 10, right: 20)
+    
+    @objc public func showcase(routes: [Route], padding: UIEdgeInsets = NavigationMapView.defaultPadding, animated: Bool = false) {
+        guard let active = routes.first,
+              let coords = active.coordinates,
+              !coords.isEmpty else { return } //empty array
+        
+        removeArrow()
+        removeRoutes()
+        removeWaypoints()
+        
+        showRoutes(routes)
+        showWaypoints(active)
+        
+        center(on: active, facing: 0, padding: padding, animated: animated)
+    }
+    
+    func center(on route: Route, facing direction:CLLocationDirection = 0, padding: UIEdgeInsets = NavigationMapView.defaultPadding, animated: Bool = false) {
```

Comment:
If it isn’t too much trouble, `fit(to:facing:edgePadding:animated:)` is a little more descriptive, since “recentering” can also describe what happens when you leave overview mode in turn-by-turn guidance (hence `recenterButton`).